### DISCUSSION TITLE
Built-in match_schema seems to ignore string patterns

### DISCUSSION THREAD

**Discussion by jimfdavies** [open-policy-agent/community#544] [OPA and Rego]
We use string match patterns to check our config values are as expected. 
We have three in-house tools, each using different JSON schema validation libraries and pattern matching works no problem.

We would like to replace these tools with OPA (or Conftest) but evaluation seems to show that 'pattern' for strings is not being caught by json.match_schema. 
I can see 'pattern' is supported by the upstream gojsonschema lib and it seems to have been carried forward into the internal opa code.

Anyway, here's a Playground to demonstrate:
https://play.openpolicyagent.org/p/ozUbc415n9

If pattern matching was used, you should see the error in the returned 'msg' document.

Please help. I don't want to do the pattern matching in Rego and want to leave that for the more nuanced policies we have.
Thanks in advance

**Comment by jimfdavies** [comment]
OK. Found this: https://github.com/open-policy-agent/opa/issues/6089

**Reply by anderseknert** [reply]
Yeah, that's an annoying issue :/
